tfhe/src/core_crypto/algorithms/polynomial_algorithms.rs

```diff
@@ -605,6 +649,7 @@ pub fn polynomial_karatsuba_wrapping_mul<Scalar, OutputCont, LhsCont, RhsCont>(
 }
 
 const KARATUSBA_STOP: usize = 64;
+const NTT_THRESHOLD: usize = 256;
```

Comment:
I don't see the specialized/hard coded test cases for the various ranges of possible Polynomial sizes ? To be sure each domain is tested